pom.xml

```diff
@@ -45,7 +45,7 @@
     <junit-bom.version>5.8.2</junit-bom.version>
     <snakeyaml.version>2.0</snakeyaml.version>
     <testcontainers.version>1.20.0</testcontainers.version>
```

Comment:
Shouldn't we use 3.3.x here instead of the latest and greatest. It is better to have the previous supported version for all our code, instead of relying on the latest Spring Boot version.

Reply:
I thought about this too.. but the end of support for is 2025-06-30. That only gives us 3 months of support. I would rather start testing 3.4.x and get ready to move to 3.5.x when it is final. I think if we are wise enough, we will align with their release cadence..